src/main/webapp/app/lecture/pdf-preview/pdf-preview.component.ts

```
@@ -230,14 +328,26 @@
-     * @param {HTMLCanvasElement} originalCanvas - The original canvas element representing the PDF page.
-     * @returns {number} The scaling factor used to resize the original canvas to fit within the container dimensions.
+     * @param originalCanvas - The original canvas element representing the PDF page.
+     * @returns The scaling factor used to resize the original canvas to fit within the container dimensions.
      */
     calculateScaleFactor(originalCanvas: HTMLCanvasElement): number {
         const containerWidth = this.pdfContainer.nativeElement.clientWidth;
         const containerHeight = this.pdfContainer.nativeElement.clientHeight;
-        const scaleX = containerWidth / originalCanvas.width;
-        const scaleY = containerHeight / originalCanvas.height;
+
+        let scaleX, scaleY;
```

Biome

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)